src/Checkbox/CheckboxLabel.js

```diff
@@ -3,6 +3,7 @@ import styled from '@emotion/styled';
 import Flex from '../Flex';
 
 const CheckboxLabel = styled(Flex.withComponent('label'))`
+  align-items: center;
```

Comment:
I believe we had this before, but applied as the prop being passed to `CheckboxLabel` since it composes the `Flex` component. Should we add it via prop again?

Reply:
Seems reasonable!